app/models/plan.rb

```diff
@@ -585,18 +579,6 @@ def landing_page
 
   private
 
-  # Initialize the title for new templates
-  #
-  # Returns nil
-  # Returns String
-  def set_creation_defaults
-    # Only run this before_validation because rails fires this before
-    # save/create
-    return if id?
-
-    self.title = "My plan (#{template.title})" if title.nil? && !template.nil?
-  end
-
```

Comment:
I'm not sure if there's a way to get this to work via an accessor as we would need a valid template which will not be available (unless specified as part of the initilizing call).
Additionally, we override this value in the `plans_controller#create` action
@briri do you know if that assumption holds for the new API, or would a default set in the accessor work for that case, as we'll have all the params at the time we make a call to `Plan.new()` ?

Reply:
The `title` is required in the new api and we reject the request if its nil or empty

Reply:
I think the Create Plan page requires the title as well. Not sure what the scenario is with copying

Reply:
Looks like the copying scenario is to prepend the title with `"Copy of " ` so this is fine to remove as it's always set at plan-creation